spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -1358,19 +1358,19 @@ describe('document-capture/components/acuant-capture', () => {
 
     expect(trackEvent.callCount).to.be.at.least(3);
     expect(trackEvent).to.have.been.calledWith('IdV: test image clicked', {
-      click_source: 'placeholder',
```

Comment:
What did I _do_ with git?

Reply:
(The actual answer: I had an incredibly-messy branch so I checked out a new one, and did `git checkout oldbranch .` and committed the result. It must not have been quite up to date. Will fix in the AM.)